keps/sig-scheduling/624-scheduling-framework/README.md

```diff
@@ -250,21 +250,31 @@ func (*BlinkingLightScorer) NormalizeScore(state *CycleState, _ *v1.Pod, nodeSco
 
 If either `Score` or `NormalizeScore` returns an error, the scheduling cycle is aborted.
 
-### Reserve
+#### Reserve
```

Comment:
Let's close #1560 as I assume everything in that PR has been covered here.

Reply:
done.